container/config.ini

```diff
@@ -18,7 +18,7 @@ cpe-oval = /usr/share/openscap/cpe/openscap-cpe-oval.xml
 ssg = /usr/share/xml/scap/ssg/content
 
 [CVEScanner]
-fetch-cve = yes
+fetch-cve = no
```

Comment:
That's a confuguration file for OpenSCAP Daemon, not for the build. You can change the setting after you install the image.